src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/organisations/OrganisationsDpsApiMockServer.kt

```diff
@@ -190,12 +191,19 @@ class OrganisationsDpsApiMockServer : WireMockServer(WIREMOCK_PORT) {
   fun pageOrganisationIdResponse(
     count: Long,
     content: List<SyncOrganisationId>,
-  ) = pageContent(
-    objectMapper = objectMapper,
-    content = content,
-    pageSize = 1L,
-    pageNumber = 0L,
+  ): String = objectMapper.writeValueAsString(
+    PagedModelSyncOrganisationId(
+      content = content,
+      page = pageMetadata(count),
+    ),
+  )
+
+  fun pageMetadata(
```

Comment:
possibly didn't need to be a separate method - but during next lot of updates (organisation word) - I will add a new pageContent method into Pagehelpers